### ISSUE TITLE
Thanks !

### ISSUE THREAD

**Issue by jaybe78** [danschultzer/elixir-terraform-aws-ecs-example#2]
Thanks for this !! It really helps !

@danschultzer Could you tell me if that type of deployment would also replicate genserver across all nodes ?

**Comment by danschultzer** [comment]
Yeah, there's node-to-node communcation, but replication will only happen if your GenServer's are set up to replicate/communicate between the nodes.
